x-pack/plugins/infra/server/graphql/log_entries/schema.gql.ts

```diff
@@ -114,5 +130,6 @@ export const logEntriesSchema = gql`
       "The query to filter the log entries by"
       filterQuery: String
     ): InfraLogSummaryInterval!
+    logItem(id: ID): InfraLogItem
```

Comment:
As mentioned above, how about making the id and return value mandatory?

```suggestion
    logItem(id: ID!): InfraLogItem!
```